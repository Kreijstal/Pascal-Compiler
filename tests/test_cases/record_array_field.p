program RecordArrayField;

type
  TStudent = record
    StudentID: integer;
    Grades: array[1..5] of integer;
  end;

var
  student1: TStudent;

begin
  student1.StudentID := 42;
  student1.Grades[3] := 95;
  writeln('The grade is: ', student1.Grades[3]);
end.
