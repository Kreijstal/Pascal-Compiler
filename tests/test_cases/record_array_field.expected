The grade is: 95
